content/doc/book/installing/servlet-containers.adoc

```diff
@@ -27,56 +27,45 @@ See the link:/doc/book/platform-information/support-policy-servlet-containers[Se
 
 WARNING: Support for traditional servlet containers may be discontinued in the future.
 
-Jenkins requires Servlet API 4.0 (Jakarta EE 8) with `javax.servlet` imports.
-Jenkins is incompatible with Servlet API 5.0 (Jakarta EE 9) or later with `jakarta.servlet` imports.
-Ensure that the Servlet API version of your chosen servlet container is compatible before running Jenkins.
-
-== Tomcat 9
-
-Tomcat 9 is based on Servlet API 4.0 (Jakarta EE 8), which is the version of the servlet API required by Jenkins.
-
-CAUTION: Later versions of Tomcat use newer versions of the Servlet API and are incompatible with Jenkins.
-
-Jenkins can be deployed to Tomcat by placing the Jenkins WAR file in the `+${CATALINA_HOME}/webapps/+` directory.
+== Requirements
+* **Jenkins 2.492.3+ requires Servlet API 5.0 (Jakarta EE 9)**
```

Comment:
```suggestion
* **Jenkins 2.492.3 and later requires Servlet API 5.0 (Jakarta EE 9)**
```
Small suggestion to make it clear that this is referring to multiple/later versions.